lib/Backend/arm64/LowerMD.h

```diff
@@ -233,7 +233,6 @@ class LowererMD
               template <bool verify = false>
               static void         Legalize(IR::Instr *const instr, bool fPostRegAlloc = false) { __debugbreak(); }
 
-              IR::Opnd*           IsOpndNegZero(IR::Opnd* opnd, IR::Instr* instr) { __debugbreak(); return 0; }
```

Comment:
I don't think we need to implement my new apis here since they're only used internally